tests/integ/test_local_mode.py

```diff
@@ -53,6 +58,22 @@ def _initialize(self, boto_session, sagemaker_client, sagemaker_runtime_client,
         self.local_mode = True
 
 
+@pytest.fixture(scope="module")
+def image_uri(
```

Comment:
call this fixture `sklearn_image_uri` since it's specific?

Reply:
Done.